sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/CheckFileLicenseTest.cs

```diff
@@ -203,10 +204,18 @@ public void CheckFileLicense_WithForcingEmptyLines_ShouldBeCompliant_CS()
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void CheckFileLicense_WhenLicensedWithMultiSingleLineCommentStartingWithNamespaceAndRegex_ShouldBeCompliant_CS()
+        public void CheckFileLicense_WhenLicensedWithMultiSingleLineCommentStartingWithNamespaceAndRegexMultiLine_ShouldBeCompliant_CS()
```

Comment:
both are RegexMultiLine - the name is "MultiSingleLineRegexHeader1"

Reply:
maybe you can allign the names of the tests with the names of the used variables holding the regex?